### ISSUE TITLE
"back" links after passing a test

### ISSUE THREAD

**Issue by cweiske** [aaronpk/micropub.rocks#62]
When a test was passed, there should be a link back to the test list.

**Comment by aaronpk** [comment]
True. For now you can click the "client tests" link in the header. It will take you back to the client you came from.
